docs/evaluation/faq/manage-datasets.mdx

```diff
@@ -10,7 +10,50 @@ import {
   TypeScriptBlock,
 } from "@site/src/components/InstructionsWithCode";
 
-# How to manage datasets programmatically
+# Managing Datasets
+
+## In LangSmith
+
+The easiest way to interact with datasets is directly in the LangSmith app. Here, you can create and edit datasets and example rows. Below are a few ways to interact with them.
+
+### From Existing Runs
+
+We typically construct datasets over time by collecting representative examples from debugging or other runs. To do this, we first filter the runs to find the ones we want to add to the dataset. Then, we create a dataset and add the runs as examples.
+
+You can do this from any 'run' details page by clicking the 'Add to Dataset' button in the top right-hand corner.
```

Comment:
Maybe we could add a screenshot of the "bulk add" here as well?

Reply:
ya - i'm gonna wait for another pr for that since i do light mode and feel like they should be consistent